packages/core/src/indexing/database-index-api.ts

```diff
@@ -29,6 +32,20 @@ export interface IndexStreamArgs {
   readonly firstAnchor: Date | null
 }
 
+/**
+ * Create a valid name for a fields index, if the index has a name specified, or generate a valid
```

Comment:
this comment is not accurate anymore as we no longer let users specify the index name